src/Plugin/Commerce/PaymentGateway/HostedPaymentFields.php

```diff
@@ -26,11 +26,11 @@
 use Symfony\Component\HttpFoundation\Request;
 
 /**
- * Provides the Bluesnap Hosted Payment Fields payment gateway.
+ * Provides the BlueSnap Hosted Payment Fields payment gateway.
  *
  * @CommercePaymentGateway(
  *   id = "bluesnap_hosted_payment_fields",
- *   label = "BlueSnap (Hosted Payment Fields)",
+ *   label = "BlueSnap Card Processing",
```

Comment:
@safalliajoseph Where exactly is this displayed, is it for store managers or for customers? I believe that store managers can customise the label displayed to the customer in the gateway form?

Reply:
yeah Its the plugin name which will be displayed to store manager when [creating the payment gateway](https://bluesnap.aws1.acrobuild.com/admin/commerce/config/payment-gateways/add)